src/server/routes/users.route.js

```diff
@@ -22,6 +22,140 @@ const router = express.Router();
  */
 router.post('/signIn', user.signin);
 
+/**
+ * @api {post} /users/edit Request Edit
+ * @apiName edit
+ * @apiGroup User
+ *
+ *  * @apiParamExample {json} Parameter Sample
+ *     {
+ *        "name": "조사라",
+ *        "gender": "female",
+ *         "email": "joytutu29@naver.com",
+ *         "languages": "Korean, English",
+ *         "location": "",
+ *         "about": "Hi, I am Sarah",
+ *         "work" : [],
+ *         "education" : [
+ *              {
+ *                  "type" : "High School",
+ *                  "school" : {
+ *                      "name" : "노은고등학교"
+ *                  }
+ *              },
+ *              {
+ *                  "type" : "College",
+ *                  "concentration" : [
+ *                      {
+ *                          "name" : "IT공학과"
+ *                      }
+ *                  ],
+ *                  "school" : {
+ *                      "name" : "숙명여자대학교 Sookmyung Women's University"
+ *                  }
+ *              }
+ *         ]
+ *     }
+ * @apiDescription {json} Success
+ *     HTTP/1.1 200 OK
+ *     {
+ *       "msg": "Update success"
+ *     }
+ * @apiErrorExample {json}
+ *     HTTP/1.1 401 Not Authenticated
+ *     {
+ *       "err_point": "Authentication failed. Please sign in first."
+ *     }
+ *
+ * @apiParam {String} name Name of User.
+ * @apiParam {String} gender Information of user gender.
+ * @apiParam {String} languages Information of user languages.
+ * @apiParam {String} location Information of user location.
```

Comment:
There's no email parameter here?
Cause I can find email field in @apiParamExample part